server/backend/api/app/crud/crud_application.py

```diff
@@ -125,7 +125,6 @@ def get_application_role_assignments(
     crossref = (
         db.query(models.FamUserRoleXref)
         .join(models.FamRole)
-        .options(joinedload('role.parent_role'))
```

Comment:
Hi Ian, just curious what is this options doing previously?

Reply:
Hi Catherine, this is an very old code and the way the joinedload was used is invalid anymore for SQLalchemy v2.
I think it is just to get the join for the parent role. I am not entirely sure if it is possible to do a `joinedload` on this case but when I review this code due to the change need from v2, I see that previous code is redundant, so I removed this. The data returned does include the parent_role (if there is) before and after removing this load. This can be verified from the swagger.

Reply:
Thanks Ian!